tests/test_paste2.py

```diff
@@ -19,85 +20,28 @@
 output_dir = test_dir / "data/output"
 
 
-def pytest_generate_tests(metafunc):
```

Comment:
I feel that metafunctions like this should only be used if the same parametrization is used in many places (let's say 3 or more). Otherwise its better to just put the decorator right where its used, reducing the need to navigate somewhere else in the codebase to see what's going on.